### DISCUSSION TITLE
Latest helm included in the container images is broken

### DISCUSSION THREAD

**Discussion by thejan2009** [helmfile/helmfile#1167] [General]
Hey,
we've been seeing some strange issues when pulling and pushing charts to OCI registries. It turns out the v3.13.2 version of Helm that's included in helmfile container image has a bunch of issues:

- https://github.com/helm/helm/issues/12491
- https://github.com/helm/helm/issues/12533

Maybe consider downgrading helm for now?

**Comment by yxxhero** [comment]
@thejan2009 waiting for fix.
